components/form/FormItem.razor.cs

```diff
@@ -574,46 +574,48 @@ ValidationResult[] IFormItem.ValidateFieldWithRules()
                 return [];
             }
 
-            if (IsRequired)
+            if (_rules?.Any() != true)
```

Comment:
Shall we keep `_rules` as `IEnumerable<FormValidationRule>`? Why not `FormValidationRule[]` or `List<FormValidationRule>`?